### ISSUE TITLE
Error while running actions

### ISSUE THREAD

**Issue by AviFix** [kiwibrowser/src.next#528]
Hi,

I forked the project and tried running the "Any branch: Build APK" action on the "kiwi" branch.

The build fails with this error
![kiwi2](https://user-images.githubusercontent.com/3835504/155955745-e8dfbb7d-ee32-4c8b-8e02-ce1af086d995.png)

The URL returns 404

I'm probably missing something very simple.

Any help would be greatly appreciated

**Comment by z8512** [comment]
Lmao

**Comment by AviFix** [comment]
> Lmao

Would you please explain?

**Comment by gnadelwartz** [comment]
To run the actions you need special permission from the owner of the original action as building requires access to GitHub build system. kiwi access keys are not cloned for security and privacy reasons.

In theory you can also build kiwi manuallyif you local linux box is equipped with fast CPUs and much RAM.
Basically it's check out original chromium source code from Google and apply the files from this repo on top.

**Comment by nift4** [comment]
> In theory you can also build kiwi manually if your local linux box is equipped with fast CPUs and much RAM.

In practice, you can't. #411

**Comment by kiwibrowser** [comment]
Yes, for now it's not possible (it was at some point), except with https://github.com/kiwibrowser/src the classic repo. The new repo still needs some work again

**Comment by AviFix** [comment]
> To run the actions you need special permission from the owner of the original action as building requires access to GitHub build system infrastucture. kiwi access keys are not cloned for security and privacy reasons.
> 
> In theory you can also build kiwi manually if your local linux box is equipped with fast CPUs and much RAM. Basically it's check out original chromium source code from Google and apply the files from this repo on top.

Thanks for the detailed explanation

**Comment by AviFix** [comment]
> Yes, for now it's not possible (it was at some point), except with https://github.com/kiwibrowser/src the classic repo. The new repo still needs some work again

So there is no way to download this repo and build it?

**Comment by AviFix** [comment]
Any news about building the project localy?

**Comment by Ibuprophen** [comment]
@AviFix, Doing this would require allot of time to explain as there's a few different avenues in approaching this. 

The following links should help you in locating information regarding this topic. 

https://docs.github.com/en/get-started/quickstart/fork-a-repo

https://frontend.turing.edu/lessons/module-1/github-repo.html

There's no "easy" way to do this in on itself. It all depends on a few factors of what you currently know, how quick of a study you can learn and the knowledge/know how of the language in which it's written (C+, Java, etc...). 

From here, you will have to do a little homework/research of your own (just 1 optional hint... Google is your friend... LOL!). 

 I wish you the best of luck! :-)) 

~Ibuprophen

**Comment by AviFix** [comment]
> https://frontend.turing.edu/lessons/module-1/github-repo.html

Hi

Thanks for your reply

The previous kiwi repo had a manual for building it locally and it worked fine for me

I'm looking for a similar manual that will explain how to build this ripo and get an APK file

**Comment by AviFix** [comment]
> Yes, for now it's not possible (it was at some point), except with https://github.com/kiwibrowser/src the classic repo. The new repo still needs some work again

Is it possible now?
Is there any way for me to do some private changes and build a custom APK?

**Comment by nullday0** [comment]
Seems like bunch of changes were just merged today: https://github.com/kiwibrowser/src.next/commit/543d5c3b5be0dd6bd84b7f2fb05618093d0494f8
Did anyone try to compile it? Does it finally have all missing resource files or it's still in non-buildable state?
Thanks.

**Comment by nift4** [comment]
I will try to compile it as it looks good on a first glance.

**Comment by nift4** [comment]
Unfortunately, it still misses the java source files specific to kiwi.
